iconv-lite-umd.d.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License.
+ *  REQUIREMENT: This definition is dependent on the @types/node definition.
+ *  Install with `npm install @types/node --save-dev`
+ *--------------------------------------------------------------------------------------------*/
+
+declare module "iconv-lite-umd" {
+  // Basic API
+  export function decode(
+    buffer: Buffer,
+    encoding: string,
+    options?: Options
+  ): string;
+
+  export function encode(
+    content: string,
+    encoding: string,
+    options?: Options
+  ): Buffer;
+
+  export function encodingExists(encoding: string): boolean;
+
+  // WARNING: Excluded because it is specific to node.
+
+  // export function decodeStream(
+  //   encoding: string,
+  //   options?: Options
+  // ): NodeJS.ReadWriteStream;
+
+  // export function encodeStream(
+  //   encoding: string,
+  //   options?: Options
+  // ): NodeJS.ReadWriteStream;
+
+  // Low-level stream APIs
+  export function getEncoder(
+    encoding: string,
+    options?: Options
+  ): EncoderStream;
+
+  export function getDecoder(
+    encoding: string,
+    options?: Options
+  ): DecoderStream;
```

Comment:
These type definitions does not exist in `0.5.1`, but the actual APIs are there. Since the author of `iconv-lite` is going to make them public in the next release anyway, I've thought it's not a problem to add them now here.

As a benefit it will untie us from `0.6` release schedule.